### PR TITLE
added the normalized example to the PointOfInteraction folder

### PR DESCRIPTION
**Pull request by Joudsi**
@jmcanterafonseca

### CODE REVIEW

File: specs/PointOfInteraction/SmartPointOfInteraction/doc/spec.md [hunk 1]
```diff
@@ -88,7 +88,72 @@ means that to test the JSON schema examples with a
...
+```json
+{
+    "id": "SPOI-ES-4326",
+    "type": "SmartPointOfInteraction",
+    "category": {
+        "value": [
+            "co-creation"
+        ]
+    }, 
+    "applicationUrl": {
```
Comment (jmcanterafonseca) [inline specs/PointOfInteraction/SmartPointOfInteraction/doc/spec.md]: "type": "url"

File: specs/PointOfInteraction/SmartPointOfInteraction/example-normalized.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (jmcanterafonseca) [inline specs/PointOfInteraction/SmartPointOfInteraction/example-normalized.json]: apply the same changes as above

File: specs/PointOfInteraction/SmartPointOfInteraction/doc/spec.md [hunk 3]
```diff
@@ -104,6 +104,7 @@ Normalized NGSI response
         ]
     }, 
     "applicationUrl": {
+        "type": "url",
```
Comment (jmcanterafonseca) [inline specs/PointOfInteraction/SmartPointOfInteraction/doc/spec.md]: should be URL

**Comment by Joudsi** [comment]
@jmcanterafonseca 
I did this fix but I am not sure if this is what you meant exactly, because I have done all the above changes.

**Comment by jmcanterafonseca** [comment]
LGTM

**Pull request merged**